src/daemon/ca/aspa.rs

```diff
@@ -0,0 +1,280 @@
+//! Autonomous System Provider Authorization
+//!
+//! This is still being discussed in the IETF. No RFC just yet.
+//! See the following drafts:
+//! https://datatracker.ietf.org/doc/draft-ietf-sidrops-aspa-profile/
+//! https://datatracker.ietf.org/doc/draft-ietf-sidrops-aspa-verification/
+//!
+
+use std::{collections::HashMap, fmt::Debug};
+
+use chrono::Duration;
+use rpki::repository::{
+    aspa::{Aspa, AspaBuilder},
+    sigobj::SignedObjectBuilder,
+    x509::Time,
+};
+
+use crate::{
+    commons::{
+        api::{AspaConfigurationUpdate, AspaCustomer, AspaDefinition, ObjectName},
+        crypto::{KrillSigner, SignSupport},
+        error::Error,
+        KrillResult,
+    },
+    daemon::{
+        ca::{AspaObjectsUpdates, CertifiedKey},
+        config::{Config, IssuanceTimingConfig},
+    },
+};
+
+pub fn make_aspa_object(
+    aspa_config: AspaDefinition,
```

Comment:
Given that there is also an `AspaConfigurationUpdate` type I find it confusing here that `aspa_config` is the name of something which is of type `AspaDefinition`. Which of the two names is "incorrect" here? config or definition?

Reply:
Using 'definition' now wherever 'configuration' was used.